Bugzilla/Install/DB.pm

```diff
@@ -822,6 +822,9 @@ sub update_table_definitions {
   $dbh->bz_add_column('components', 'team_name',
     {TYPE => 'varchar(255)', NOTNULL => 1, DEFAULT => "'Mozilla'"});
 
+  # Bug XXXX - dkl@mozilla.com
+  $dbh->bz_add_column('logincookies', 'auth_type',{TYPE => 'varchar(256)'});
```

Comment:
`256` is very large for our needs; after switching to storing the login provider's name update this column's size accordingly (here and in the schema def).